### PR TITLE
Swedish fixes

### PR DESCRIPTION
**Pull request by squid-box**
Found a few missing translations, some small errors, and then a bit of rewriting to align everything with corrections and new translations.

I tried to separate it all so any iffy parts can be excluded. I'm not sure I got the Twitter text-concatenation parts 100% perfect.

### CODE REVIEW

**Comment by RaMMicHaeL** [comment]
Thank you for the updates! I will merge the poll request soon.

> I'm not sure I got the Twitter text-concatenation parts 100% perfect.

You can look at tweet examples [here](https://twitter.com/search?f=tweets&vertical=default&q=%23anti_pup&src=typd).

@dotar, could you please review the changes? Thanks.

File: program/Swedish (Sweden)/app.txt [hunk 1]
```diff
@@ -7,19 +7,19 @@ IDS_MAIN_WNDTITLE       "Unchecky"
 IDS_HEADER_MAIN         "&Hem"
 IDS_HEADER_CONFIG       "&Inställningar"
 IDS_HEADER_ABOUT        "&Om"
-IDS_MAINCTRL_TEXT       "Välkommen till Unchecky, programmet som har som försöker hålla din dator fri från oönskade program."
-IDS_MAINCTRL_TEXT_SVC_RUNNING "Tjänsten är igång."
+IDS_MAINCTRL_TEXT       "Välkommen till Unchecky, programmet som försöker hålla din dator fri från oönskade program."
+IDS_MAINCTRL_TEXT_SVC_RUNNING "Unckecky-tjänsten körs."
 IDS_MAINCTRL_TEXT_SVC_STOPPED 
-                        "Tjänsten är avbruten. Tjänsten kommer återupptas vid omstart av datorn."
+                        "Unckecky-tjänsten är stoppad. Tjänsten kommer återupptas vid omstart av datorn."
```
Comment (dotar) [inline program/Swedish (Sweden)/app.txt]: "Unchecky-tjänsten" is misspelled.

File: program/Swedish (Sweden)/setup.txt [hunk 2]
```diff
@@ -14,7 +14,7 @@ IDS_EXTRACTRL_TEXT      "Unchecky installeras i följande mapp. För att install
 IDS_EXTRACTRL_FOLDER_GROUP "&Installationsmapp"
 IDS_EXTRACTRL_FOLDER_BROWSE "&Bläddra..."
 IDS_EXTRACTRL_BROWSE_TEXT "Välj den mapp där Unchecky ska installeras:"
-IDS_FINISHCTRL_TITLE    "Tjänsten är igång"
+IDS_FINISHCTRL_TITLE    "Gratulerar! Unchecky har installerats på din dator.\n\nUnckecky-tjänsten kommer kontinuerligt köras i bakgrunden, och bevaka dina installationer. Du kan starta Unchecky när som helst från skrivbordsgenvägen för att se hur många kryssrutor som urkryssats."
```
Comment (dotar) [inline program/Swedish (Sweden)/setup.txt]: The wrong line is translated. This should be the translation for `IDS_FINISHCTRL_TEXT` (and notice that `\n\nUnckecky-tjänsten.` is misspelled).
The translation for `IDS_FINISHCTRL_TITLE` should be "Unchecky-tjänsten är igång!"

File: program/Swedish (Sweden)/app.txt [hunk 3]
```diff
@@ -49,19 +49,19 @@ IDS_LOG_SHARE_FACEBOOK  "Dela på Facebook"
...
 IDS_LOG_CLEAR_WARNING   "Aktivitetsloggen kommer rensas. Är du säker på att du vill fortsätta?"
-IDS_SHARE_EMAIL_SUBJECT "A Program Recommendation: Unchecky"
-IDS_SHARE_EMAIL_MAIN    "Hi!\n\nI want to recommend you a great program I've stumbled upon recently.\nIt's called Unchecky and it helps prevent potentially unwanted programs from being installed on your computer.\n%1You can get it here:\nhttp://unchecky.com/?ref=email\n\nTake care."
-IDS_SHARE_EMAIL_EXAMPLE "For example, it %1 while installing the following program: %2.\n"
-IDS_SHARE_EMAIL_PREVENTED "prevented %u potentially unwanted programs"
-IDS_SHARE_EMAIL_AND     " and "
-IDS_SHARE_EMAIL_WARNED  "warned me about %u offers"
+IDS_SHARE_EMAIL_SUBJECT "Programrekommendation: Unchecky"
+IDS_SHARE_EMAIL_MAIN    "Hej!\n\nJag vill rekommendera ett program jag precis hittat.\nDet heter Unchecky hjälper dig att hindra potentiellt oönskade program från att installeras på din dator.\n%1Du kan få tag på det här:\nhttp://unchecky.com/?ref=email\n\nTa hand om dig!."
+IDS_SHARE_EMAIL_EXAMPLE "Till exempel, det %1 medans jag installerade: %2.\n"
```
Comment (dotar) [inline program/Swedish (Sweden)/app.txt]: This should be `Till exempel %1 medans jag installerade: %2.\n`.

File: program/Swedish (Sweden)/app.txt [hunk 4]
```diff
@@ -49,19 +49,19 @@ IDS_LOG_SHARE_FACEBOOK  "Dela på Facebook"
...
+IDS_SHARE_EMAIL_PREVENTED "hindrade %u potentiellt önskade program"
```
Comment (dotar) [inline program/Swedish (Sweden)/app.txt]: This should be `hindrades %u potentiellt önskade program`.

File: program/Swedish (Sweden)/app.txt [hunk 5]
```diff
@@ -49,19 +49,19 @@ IDS_LOG_SHARE_FACEBOOK  "Dela på Facebook"
...
+IDS_SHARE_EMAIL_AND     " och "
+IDS_SHARE_EMAIL_WARNED  "varnade mig om %u erbjudanden"
```
Comment (dotar) [inline program/Swedish (Sweden)/app.txt]: This should be `jag blev varnad om %u erbjudanden`.

**Comment by dotar** [comment]
@RaMMicHaeL I've reviewed the changes and made comments where appropriate.

File: program/Swedish (Sweden)/app.txt (hunk 1 above)
Comment (squid-box) [inline program/Swedish (Sweden)/app.txt]: Fixed in commit 6280677.

File: program/Swedish (Sweden)/setup.txt (hunk 2 above)
Comment (squid-box) [inline program/Swedish (Sweden)/setup.txt]: Fixed in commit 6280677.

**Comment by squid-box** [comment]
@dotar: As I prefaced my pull request with, I wasn't entirely sure how the Twitter/Email concatenation thing worked. To me it looks like the resulting message (%1) could contain either the text about unwanted programs, the text about warnings, or both. My suggested translation would end up with the possible texts:

1. "Till exempel, det hindrade %u potentiellt önskade program och varnade mig om %u erbjudanden medans jag installerade: <program>."
2. "Till exempel, det hindrade %u potentiellt önskade program medans jag installerade: <program>."
3. "Till exempel, det varnade mig om %u erbjudanden medans jag installerade: <program>."

And your suggested fixes would make this:

1. "Till exempel hindrades %u potentiellt önskade program och jag belv varnad om %u erbjudanden medans jag installerade: <program>."
2. "Till exempel hindrades %u potentiellt önskade program medans jag installerade: <program>."
3. "Till exempel jag belv varnad om %u erbjudanden medans jag installerade: <program>."

The third text in your case would be less correct (proper sentence should be "jag blev ..." rather than "blev jag ...", even if the meaning comes through.

( And yes, I see the "potentiellt önskade" to "potentiellt oönskade" and "belv" to "blev" :) )

**Comment by RaMMicHaeL** [comment]
Thank you for the through review @dotar!

@squid-box, thanks for the fixes. [The website](https://unchecky.com/) was updated with the new translation, and the updated program translation will be available in the next version of Unchecky.

**Comment by dotar** [comment]
@squid-box: Ok, I agree on the lines regarding email. "Till exempel, det hindrade [...]" looked wrong grammatically at first, but I guess the comma makes it better ;)

**Pull request merged**